index.html

```diff
@@ -5,43 +5,92 @@
+                </ul>
+                <img id="basket" class="basket" src="assets/basket.svg" />
             </div>
 
             <div class="stats-box">
-                <div class="stats-row">
-                    <span class="label">Score:</span>
-                    <span id="score-value" class="value" />
+                <h1>Rocket launcher</h1>
+
+                <div class="stats-row-center">
+                    <span id="score-value" class="value">10</span>
+                    <span class="label">Score</span>
                 </div>
-                <div class="stats-row">
-                    <span class="label">Level:</span>
-                    <span id="level-value" class="value" />
+                <div class="stats-row-center">
+                    <span id="level-value" class="value">2</span>
+                    <span class="label">Level</span>
                 </div>
                 <div class="stats-row">
                     <span class="label">Lives:</span>
-                    <span id="lives-remaining-value" class="value" />
+                    <span id="lives-remaining-value" class="value">33</span>
+                </div>
+                <div class="text-block">
+                    <p class="title">Instructions</p>
+                    <p class="description">
+                        Lorem ipsum dolor sit, amet consectetur adipisicing
+                        elit. Reprehenderit ipsa voluptatum quae adipisci
+                        doloremque autem officia dolor amet quas, dolore modi
+                        magni quis quo sunt rerum ducimus soluta. Sequi,
+                        suscipit!
+                    </p>
                 </div>
-                <!-- <div class="stats-row">
-                    <a class="label">How to play</a>
-                </div> -->
             </div>
         </div>
-        <div id="game-overlay">
-            <span id="overlay-text-start">Press Enter to play again</span>
-            <span id="overlay-text-pause">Press Enter to play again</span>
-            <span id="overlay-text-gameover">Press Enter to play again</span>
+        <div id="game-overlay" class="overlay">
+            <div class="overlay-content">
+                <span id="overlay-text-start" class="overlay-text"
+                    >Press Enter to play again</span
+                >
+                <span id="overlay-text-pause" class="overlay-text"
+                    >Press Enter to play again</span
+                >
+                <span id="overlay-text-gameover" class="overlay-text"
+                    >Press Enter to play again</span
+                >
+            </div>
         </div>
     </body>
-    <script src="./stateMutators.js"></script>
+    <script type="module" src="./stateMutators.js"></script>
     <script type="module" src="./EventListeners.js"></script>
-
+    <script type="module" src="./rendering.js"></script>
+    <script type="module" src="./main.js"></script>
```

Comment:
Do we need to add script tags for all of these modules or just the base module, considering that the other modules are `import`ed from there?

Easy to change anyway, just not sure how it works.

Reply:
I've just tried, you don't have to add all the script tag there, only the main one. I will remove others